### PR TITLE
rxjs-no-implicit-any-catch

### LINKED ISSUE #779: rxjs-no-implicit-any-catch
### 💥 Missing Converter

-   `tslint-to-eslint-config` version: 1.5.0
-   `typescript-eslint` version: 4.6.0

#### TSLint Rule Name
rxjs-no-implicit-any-catch

#### Package Source
rxjs-tslint-rules

### PR DESCRIPTION
**Pull request by criskrzysiu**
## PR Checklist

-   [x] Addresses an existing issue: fixes #779
-   [x] That issue was marked as [`status: accepting prs`](https://github.com/typescript-eslint/tslint-to-eslint-config/labels/status%3A%20accepting%20prs)

## Overview

rxjs-no-implicit-any-catch => [rxjs/no-implicit-any-catch](https://github.com/cartant/eslint-plugin-rxjs/blob/main/docs/rules/no-implicit-any-catch.md)

### CODE REVIEW

File: src/converters/lintConfigs/rules/ruleConverters/eslint-plugin-rxjs/no-implicit-any-catch.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { RuleConverter } from "../../ruleConverter";
+
+export const convertNoImplicitAnyCatch: RuleConverter = () => {
+    return {
+        rules: [
+            {
+                ruleName: "rxjs/no-implicit-any-catch",
+            },
+        ],
```
Comment (KingDarBoja) [inline src/converters/lintConfigs/rules/ruleConverters/eslint-plugin-rxjs/no-implicit-any-catch.ts:9]: This is missing the rule arguments as stated on https://github.com/cartant/rxjs-tslint-rules/blob/master/source/rules/rxjsNoImplicitAnyCatchRule.ts

**Review by KingDarBoja — CHANGES_REQUESTED** [review]

File: src/converters/lintConfigs/rules/ruleConverters/eslint-plugin-rxjs/no-implicit-any-catch.ts (hunk 1 above)
Comment (criskrzysiu) [reply src/converters/lintConfigs/rules/ruleConverters/eslint-plugin-rxjs/no-implicit-any-catch.ts:9]: Good catch, there was no mention of this parameter on the tslint rules description page. https://github.com/cartant/rxjs-tslint-rules. 
Will fix it.

Comment (criskrzysiu) [reply src/converters/lintConfigs/rules/ruleConverters/eslint-plugin-rxjs/no-implicit-any-catch.ts]: @KingDarBoja fixed

**Review by KingDarBoja — APPROVED** [review]

**Comment by KingDarBoja** [comment]
Harder, better, faster, stronger 🤖

**Pull request merged**